### PR TITLE
dev: remove information refering to Mumble server, add document point…

### PR DESCRIPTION
**Pull request by peterbarker**
…ing to Discord

### CODE REVIEW

File: dev/source/docs/ardupilot-discord-server.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.. _ardupilot-discord-server:
+
+========================
+ArduPilot Discord Server
+========================
+
+The ArduPilot dev team uses `Discord <https://ardupilot.org/discord>`__ for real-time voice chat.  We have a "General Discussion" channel for chatting about ArduPilot development at any time. How many developers are connected depends a lot on timezones.  Don't be shy about your English language ability, we have developers from all over the world.
```
Comment (rmackay9) [inline dev/source/docs/ardupilot-discord-server.rst:7]: Is the main discord channel only for development?  It seems to just be a huge chat..

**Comment by rmackay9** [comment]
Really great to see this PR!

File: dev/source/docs/ardupilot-discord-server.rst (hunk 1 above)
Comment (peterbarker) [reply dev/source/docs/ardupilot-discord-server.rst:7]: I think we decided this was OK as-is via... Discord :-)

File: dev/source/docs/ardupilot-mumble-server.rst [hunk 2]
```diff
@@ -4,43 +4,6 @@
...
-Time of the meeting is: 2300 UTC
-A calendar with the meeting can be found at https://calendar.google.com/calendar/embed?src=rgdbom27tb1vlo62kjjnmt8va4%40group.calendar.google.com which will show the correct time for you timezone.
-
-The agenda and minutes are normally sent to discuss.ardupilot.org.
-
-Prefer Typing?
---------------
-
-If you prefer text chat, please join us on `gitter <https://gitter.im/ArduPilot/ardupilot>`__
+We're now using `ArduPilot Discord (live chat) <https://ardupilot.org/discord>`__ - but please direct support questions in the first instance to instance to `our Discuss forums <https://discuss.ardupilot.org/>`__
```
Comment (Hwurzburg) [inline dev/source/docs/ardupilot-mumble-server.rst:9]: "...direct support questions first to .....

File: dev/source/docs/how-the-team-works.rst [hunk 3]
```diff
@@ -18,4 +18,4 @@ These pages are less about the software directly and more about the procedures a
     Roadmap <roadmap>
     Top Contributors <common-team>
     Voting <how-the-team-works-voting>
-    Weekly meetings on Mumble Server <ardupilot-mumble-server>
```
Comment (Hwurzburg) [inline dev/source/docs/how-the-team-works.rst:21]: must move to archived topics document to keep in TOC

File: dev/source/docs/ardupilot-mumble-server.rst [hunk 4]
```diff
@@ -4,43 +4,6 @@
 ArduPilot Mumble Server
 =======================
```
Comment (Hwurzburg) [inline dev/source/docs/ardupilot-mumble-server.rst:6]: add " .. note:: Archived Topic"

Comment (peterbarker) [reply dev/source/docs/ardupilot-mumble-server.rst:6]: Done

File: dev/source/docs/ardupilot-mumble-server.rst (hunk 2 above)
Comment (peterbarker) [reply dev/source/docs/ardupilot-mumble-server.rst]: Fixed.

File: dev/source/docs/how-the-team-works.rst (hunk 3 above)
Comment (peterbarker) [reply dev/source/docs/how-the-team-works.rst:21]: Done.

**Comment by Hwurzburg** [comment]
thanks! merged

**Pull request merged**